apps/server/src/modules/learnroom/uc/course-copy.uc.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { Configuration } from '@hpi-schul-cloud/commons';
-import { AuthorizationContextBuilder } from '@modules/authorization';
-import { AuthorizableReferenceType, AuthorizationReferenceService } from '@modules/authorization/domain';
+import { AuthorizableReferenceType, AuthorizationContextBuilder } from '@modules/authorization';
 import { CopyElementType, CopyStatusEnum } from '@modules/copy-helper';
 import { ForbiddenException, InternalServerErrorException } from '@nestjs/common';
 import { Test, TestingModule } from '@nestjs/testing';
 import { Permission } from '@shared/domain/interface';
 import { courseFactory, setupEntities, userFactory } from '@shared/testing';
+import { AuthorizationReferenceService } from '@src/modules/authorization-reference';
```

Comment:
Hat auch nichts mit dem PR zu tun, aber das haben wir beim aufräumen übersehen. 
Der Reference Service sollte hier gar nicht notwendig sein und reduziert vermutlich auch deutlich die performance der Operation. Das müsste noch umgebaut werden. Wenn nicht als Teil dieses PRs, dann brauchen wir zumindest ein Ticket um AuthorizationReferenceService in AuthorizationService zu ändern.

Edit: gilt natürlich eigentlich nicht primär für das spec file, sondern den prod code. :)